### PR TITLE
decompile tevutil

### PR DESCRIPTION
**Pull request by chmcl95**
decompile tevutil.s (except 3 functions)

about excepted 3 functions
GXSetTevColorOp_cached_init : almost decompiled. but less parameters on GXSetTevColorOp.
GXSetTevAlphaOp_cached_init : almost decompiled. but less parameters GXSetTevAlphaOp.
GXSetTevOrder_cached : almost decompiled. but order is wrong.

### CODE REVIEW

**Comment by chmcl95** [comment]
decompile tevutil.s (except 2 functions)

about excepted 2 functions
GXSetTevColorOp_cached_init : almost decompiled. but less parameters on GXSetTevColorOp.
GXSetTevAlphaOp_cached_init : almost decompiled. but less parameters GXSetTevAlphaOp.

**Comment by chmcl95** [comment]
decompile tevutil.s. (100% done!)

This pull request also changes, "GXPixel.h" and "GXTev.h" .

File: include/dolphin/GXPixel.h [hunk 1]
```diff
@@ -3,9 +3,29 @@
...
+// 0x800E308C
+// ? GXSetPixelFmt();
+// 0x800E319C
+void GXSetDither(GXBool dither);
+// 0x800E31DC
+// ? GXSetDstAlpha();
+// 0x800E3230
+// ? GXSetFieldMask();
+// 0x800E3268
+// ? GXSetFieldMode();
```
Comment (alexozer) [inline include/dolphin/GXPixel.h:29]: Should you remove these commented lines?

File: src/tevutil.c [hunk 2]
```diff
@@ -0,0 +1,690 @@
...
+        (gxCache->blendMode).src_factor = src_factor;
+        (gxCache->blendMode).dst_factor = dst_factor;
+    }
+    return;
+}
+
+void GXSetBlendMode_cached_init(GXBlendMode type, GXBlendFactor src_factor, GXBlendFactor dst_factor, GXLogicOp op)
+{
+    GXSetBlendMode(type, src_factor, dst_factor, op);
+    (gxCache->blendMode).type = type;
```
Comment (camthesaxman) [inline src/tevutil.c]: These parentheses here are probably not necessary. If it still matches, you can get rid of them.
`gxCache->blendMode.type = type;`

Comment (chmcl95) [reply src/tevutil.c]: I confirmed It's still matching.
Fixed pick up codes.
And fix similar codes.

**Pull request merged**